commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/display/context/CPCompareContentMiniDisplayContext.java

```diff
@@ -247,7 +247,7 @@ public boolean isSelectionStyleCustomRenderer() {
 	private final CPContentListRendererRegistry _cpContentListRendererRegistry;
 	private final CPDefinitionHelper _cpDefinitionHelper;
 	private final List<Long> _cpDefinitionIds;
-	private final CPRequestHelper _cpRequestHelper;
+	private final CPContentRequestHelper _cpRequestHelper;
```

Comment:
`CPContentRequestHelper _cpRequestHelper` should be `CPContentRequestHelper _cpContentRequestHelper`